ccdproc/tests/test_ccdproc.py

```diff
@@ -481,6 +481,47 @@ def test_flat_correct_min_value(ccd_data):
     assert flat_orig_data is not flat.data
 
 
+# test for flat correction
+@pytest.mark.data_scale(10)
+def test_flat_correct_mean_value(ccd_data):
+    # add metadata to header for a test below...
+    ccd_data.header['my_key'] = 42
+    size = ccd_data.shape[0]
+    # create the flat, with some scatter
+    # Note that mean value of flat is set below and is different than
+    # the mean of the flat data.
+    flat_mean = 5.0
+    data = np.random.normal(loc=1.0, scale=0.05, size=(size, size))
+    flat = CCDData(data, meta=fits.header.Header(), unit=ccd_data.unit)
```

Comment:
you can omit the `.header`, just using `fits.Header` should work without problems.